whitepaper/chapters/cryptography.tex

```diff
@@ -6,17 +6,17 @@ \section{Cryptography}
 }{Whitfield Diffie}
 
 \nemchapterfirstletter{B}{lockchain} technology demands the use of some cryptographic concepts.
-\codenamespace is using cryptography based on Elliptic Curve Cryptography (ECC).
+\codenamespace uses cryptography based on Elliptic Curve Cryptography (ECC).
 The choice of the underlying curve is important in order to guarantee security and speed.
 
-\codenamespace has chosen to use the \emph{Twisted Edwards curve}:
+The \codenamespace team chose the \emph{Twisted Edwards curve}:
```

Comment:
would prefer to drop references to "team"

The specific curve being used is the ...